addons/binding/org.openhab.binding.robonect/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,256 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions
+		bindingId="robonect" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+		xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="mower">
+		<label>Mower</label>
+		<description>Mower robot connected via robonect module</description>
+		<channels>
+			<channel id="name" typeId="nameType"/>
+			<channel id="battery" typeId="system.battery-level"/>
+			<channel id="wlan-signal" typeId="system.signal-strength"/>
+			<channel id="mowing-hours" typeId="hoursType"/>
+
+			<channel id="mode" typeId="modeType"/>
+			<channel id="started" typeId="startedType"/>
```

Comment:
Is this about the "start" command (http://www.robonect.de/viewtopic.php?f=11&t=39)?
If so, I would simply suggest to name it "start" - this would be in line with the module API and also not appear to be read-only for the Thing.

Reply:
:-) Thank you. So simple but logic. Somehow I was always looking at the name of the status message and thus took the wrong base. So finally simply renamed it to "start" as you suggested.